examples/tv-app/tv-common/shell/AppTvShellCommands.cpp

```diff
@@ -277,6 +282,63 @@ static CHIP_ERROR AppPlatformHandler(int argc, char ** argv)
 
         return CHIP_NO_ERROR;
     }
+    else if (strcmp(argv[0], "printclients") == 0)
+    {
+        if (argc < 2)
+        {
+            return PrintAllCommands();
+        }
+        char * eptr;
+
+        uint16_t endpoint = (uint16_t) strtol(argv[1], &eptr, 10);
```

Comment:
please consider to remove error-prone strtol usage, replace with std::from_chars

Reply:
leaving this in for now to remain consistent with the rest of this file (and minimize c++17 dependencies for now)